common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -286,10 +287,38 @@ def _get_user(self):
         """
         Return the current runtime Django user.
         """
-        from django.contrib.auth.models import User
-        return User.objects.get(id=self.runtime.user_id)
+        return get_user_model().objects.get(id=self.runtime.user_id)
 
-    def gate_sequence_if_it_is_a_timed_exam_and_contains_content_type_gated_problems(self):
+    def _check_children_for_content_type_gating_paywall(self, item):
+        """
+        If:
+        This xblock contains problems which this user cannot load due to content type gating
+        Then:
+        Return the first content type gating paywall (Fragment)
+        Else:
+        Return None
+        """
+        try:
+            user = self._get_user()
+            course_id = self.runtime.course_id
+            content_type_gating_service = self.runtime.service(self, 'content_type_gating')
+            if not (content_type_gating_service and
+                    content_type_gating_service.enabled_for_enrollment(user=user, course_key=course_id)):
+                self.gated_sequence_paywall = None
+                return None
```

Comment:
Nit: `return None` -> `return`

Reply:
curious if there is any style guide suggestions on which is better? I don't have a strong feeling either way but looking at our code including this very file we seem to have a mix
(personally, I think return None might be clearer if it is a little more explicit)

Reply:
Yeah, it's a nit. I don't think we have any set style guide rule so I'm happy with it either way.